### PR TITLE
check whether bplan points match - on hold

### PR DESCRIPTION
**Pull request by der-john**

### CODE REVIEW

File: bplan/management/commands/load_bplan.py [hunk 1]
```diff
@@ -230,7 +246,8 @@ def handle(self, *args, **options):
             if not geometry.valid:
                 invalid_items.append(planname)
 
-            point = self._calculate_point(multipolygon)
+            point = self._calculate_point(multipolygon, points)
+            points.append(point)
```
Comment (der-john) [inline bplan/management/commands/load_bplan.py]: I know this is data duplication. But for some reason, I couldn't do `Bplan.objects.all()` inside the script. :(

**Comment by MagdaN** [comment]
@der-john thanks for the PR, I will look at it but I have one question: Did you consider that the points have to have some distance from each other, because on the map there will be markers, which need to be clicked?

**Comment by der-john** [comment]
that's a very valid question: what do you think is a minimum distance / how do we find out what it should be?

**Comment by der-john** [comment]
I'd like to check how close the points now lie and am waiting for @MagdaN to create a lookup function to locate bplan points on the map. Until then this is on hold.

**Comment by der-john** [comment]
I'll avoid the conflicts and do this from scratch.

**Pull request closed without merging**